pkg/gadgets/audit-seccomp/tracer/bpf/audit-seccomp.c

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright (c) 2021 The Inspektor Gadget authors */
+
+/* This BPF program uses the GPL-restricted function bpf_probe_read*().
+ */
+
+#include <vmlinux/vmlinux.h>
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_tracing.h>
+
+#include "audit-seccomp.h"
+
+struct {
+	__uint(type, BPF_MAP_TYPE_PERF_EVENT_ARRAY);
+} events SEC(".maps");
+
+SEC("kprobe/audit_seccomp")
```

Comment:
Out of curiosity, is there something like this?
```suggestion
SEC("kfunc/audit_seccomp")
```

Reply:
I don't see support for kfunc in the cilium/ebpf library that we use in Inspektor Gadget.